### PR TITLE
[Template Security]LPS-134874

### PR DESCRIPTION
**Pull request by tinatian**

### CODE REVIEW

**Comment by tinatian** [comment]
ci:test:relevant

**Comment by tinatian** [comment]
ci:test:sf

**Comment by tinatian** [comment]
ci:test:relevant

**Comment by tinatian** [comment]
ci:test:sf

File: portal-impl/src/com/liferay/portal/template/ServiceLocator.java [hunk 1]
```diff
@@ -41,6 +66,12 @@ public Object findService(String servletContextName, String serviceName) {
 	private ServiceLocator() {
 	}
 
+	private static final boolean _TEMPLATE_ENGINE_SERVICE_LOCATOR_RESTRICT =
```
Comment (shuyangzhou) [inline portal-impl/src/com/liferay/portal/template/ServiceLocator.java:73]: @tinatian this is in portal-impl, the props should be declared in PropsValues. Please add a field there.

**Comment by shuyangzhou** [comment]
@tinatian see the comment above, please resend, thanks!

File: portal-impl/src/com/liferay/portal/template/ServiceLocator.java (hunk 1 above)
Comment (tinatian) [reply portal-impl/src/com/liferay/portal/template/ServiceLocator.java:73]: @shuyangzhou It is only used once in this class, why have to declare in PropsValues? I remember we do have properties without definition in PropsValues.

Comment (shuyangzhou) [reply portal-impl/src/com/liferay/portal/template/ServiceLocator.java:73]: We normally only do that when usage is inside kernel. It is not a strictly enforced rule. There might be exceptions. But when the usage is inside impl, keep it inside PropsValues to follow the common style.

**Pull request closed without merging**